docs/contributors/github-labels.md

```diff
@@ -64,6 +64,7 @@ The labels in this list originated within Kubernetes at
 | area/proxy/clusterip               | Issues or PRs related to the implementation of ClusterIP Services | Any |
 | area/proxy/nodeport                | Issues or PRs related to the implementation of NodePort Services | Any |
 | area/proxy/nodeportlocal           | Issues or PRs related to the NodePortLocal feature | Any |
+| area/secondary-networks            | Issues or PRs related to support for secondary networks in Antrea | Any |
```

Comment:
Just use `secondary-network`? No plural in the list.

Reply:
Done